Http/PjaxRedirectResponse.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * This file is part of the eZ PlatformUI package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\PlatformUIBundle\Http;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Specific redirect response, dedicated to PlatformUI JS app.
+ */
+class PjaxRedirectResponse extends Response
```

Comment:
to clarify a bit the usage of this kind of response, I would rather call it something like PjaxFormProcessingDone (and then also change the name of the methods in the abstract controller class) as the redirection is purely a behaviour of JavaScript application (different layer)